strawberryfields/backends/fockbackend/tests/test_fock_probs_gaussian_states.py

```diff
@@ -0,0 +1,173 @@
+# Copyright 2019 Xanadu Quantum Technologies Inc.
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+r"""Unit tests for probabilities of Fock outcomes of Gaussian states"""
+import numpy as np
+from scipy.special import factorial as fac
+
+NUM_REPEATS = 50
+
+
+def squeezed_matelem(r, n):
```

Comment:
This function is not being used anywhere (my fault for accidentally porting it), can be deleted.